src/components/EditorTopBar.tsx

```diff
@@ -43,14 +44,7 @@ export const EditorTopBar = ({
         <li>
           <h2>{title}</h2>
         </li>
-        <li>
-          <button
-            onClick={onSave}
-            className="dp-button button-medium primary-green"
-          >
-            <FormattedMessage id="save" />
-          </button>
-        </li>
+        {children ? <li>{children}</li> : ""}
```

Comment:
There should be an `<li>` for each _child_ or all the buttons should share the same `<li>`?

CC: @GusBaamonde

Reply:
I like to individualize them because it gives us versatility. We can move them all together from the ul or with very little effort space them equally or add a class and manipulate a single particular button